src/mobile/multichoice/test/items.test.ts

```diff
@@ -0,0 +1,142 @@
+import { expect, fixture } from "@open-wc/testing";
+import { MobileMultiChoice } from "../index";
+
+describe("MobileMultiChoice", () => {
```

Comment:
@nmanhit 
Please check all cases of this files:
+ There is missing item label expectation
+ It will better if replacing loop (for):
Ex: 
```
      const itemsEl = el.getElementsByTagName("option");
      expect(itemsEl.length).to.be.equal(3);
      for (let i = 0; i < itemsEl.length; i++) {
        const itemEl = itemsEl[i] as HTMLElement;
        const value = itemEl.getAttribute("value")?.trim();
        expect(value).to.have.equal(expectedValues[i]);
      }
```
Replace to:
```
     const itemsEl = el.getElementsByTagName("option");
     expect(itemsEl[0].getAttribute("value")?.trim()).to.have.equal(expectedValues[0]);
     expect(itemsEl[0].textContent?.trim()).to.have.equal(expectedLabels[0]);

     expect(itemsEl[1].getAttribute("value")?.trim()).to.have.equal(expectedValues[1]);
     expect(itemsEl[1].textContent?.trim()).to.have.equal(expectedLabels[1]);

     expect(itemsEl[2].getAttribute("value")?.trim()).to.have.equal(expectedValues[2]);
     expect(itemsEl[2].textContent?.trim()).to.have.equal(expectedLabels[2]);
```

+ Please add TODO for case that set items to null and dupplicate values.